### ISSUE TITLE
`no-zero-fractions`: Not working well with numeric separators

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1224]
```js
const foo = 123.11100000_000
```

```js
const foo = 1_2.
```

Should be reported, but they are not.

[`no-zero-fractions`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-zero-fractions.md)

**Comment by fisker** [comment]
@sindresorhus Which one do you prefer?

1.
  ```js
  1..toString()
  ```
 
2. My choice

  ```js
  (1).toString()
  ```


We fixed to broken code now.

**Comment by sindresorhus** [comment]
`2`. I would even go as far as to suggest a rule that disallows `1`.

**Comment by fisker** [comment]
`1..toString()` will fix to `(1).toString()`, no need new rule. But `1 .toString()` won't be checked.
